src/checkout/components/Totals/Totals.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from "react";
+import { TotalRow } from "../TotalRow";
+import { useTranslate, Stack } from "../../../_shared";
+import {
+  calculateVat,
+  calculateShipping,
+  calculateGrandTotal,
+} from "../../utils";
+
+interface Props {
+  total: number;
+}
+
+export const Totals: FC<Props> = ({ total }) => {
+  const translate = useTranslate();
+
+  return (
+    <Stack direction="column" spacing="8px">
```

Comment:
mui has a spacing system that uses numbers as identifier of theme dimensions: for ui consistency it is better to use them instead of raw pixels, so it can be controlled directly from the theme. 8 is 1, 16 is 2 etc